en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
 
 ## Changes
 
+* [Common](#common)
+* [MAVLink](#mavlink)
+* [Commander](#commander)
 * [Multicoper](#multicopter)
+* [VTOL](#vtol)
+* [Control & Navigation](#control)
+
+### Common
+
+* **RTL Trigger based on remaining flight range ([PR#16399](https://github.com/PX4/PX4-Autopilot/pull/16399))**
+  * Calculates time to home, on RTL, taking into account vehicle speed, wind speed, and desination distance/direction
+* **Pre-emptive geofence breach ([PR#16400](https://github.com/PX4/PX4-Autopilot/pull/16400))**
+  * Triggers a breach if the _predicted_ current trajectory will result in a breach, allowing the vehicle to be re-routed to a safe hold position.
+
+### MAVLink
+
+* **MAVLink Ethernet configuration ([PR#14460](https://github.com/PX4/PX4-Autopilot/pull/14460))**
+  * MAVLink Ethernet channel settings such as UDP port, remote port and broadcast mode now can be changed dynamically via parameters.
+* **Support for COMPONENT_INFORMATION ([PR#16039](https://github.com/PX4/PX4-Autopilot/pull/16039))**
+  * This new message enables a GCS to understand which commands are supported by any given Autopilot (RFC: [mavlink#1339](https://github.com/mavlink/mavlink/issues/1339))
+
+### Commander
+
+* **Commander: use control mode flags and cleanup arm_disarm ([PR#16266](https://github.com/PX4/PX4-Autopilot/pull/16266))**
+  * Consolidates arming logic, and does a better job at keeping the current state
```

Comment:
What does "better job at keeping current state" mean?

Essentially as a user, why do I care about using these flags and this cleanup? Do I have to do anything different? What will I see different? I.e. might I get better information on why things won't arm in the GCS?

Reply:
I think this is not an end-user feature, this is geared towards developers, I will reword to add better detail, explaining the changes